docs/LibraryEvolution.rst

```diff
@@ -513,29 +487,29 @@ clients to access them more efficiently. This restricts changes a fair amount:
 .. admonition:: TODO
 
     It Would Be Nice(tm) to allow marking the *getter* of a top-level variable
-    inlineable while still allowing the setter to change. This would need
+    inlinable while still allowing the setter to change. This would need
     syntax, though.
 
-Any inlineable accessors must follow the rules for `inlineable functions`_, as
+Any inlinable accessors must follow the rules for `inlinable functions`_, as
 described above.
 
-Unlike functions, inlineable stored constants and variables (including those
-with accessors) may not be removed even if they have been inlineable since they
-were first introduced, because the storage for these bindings is still part of
-the library in which they were defined. Removing computed variables that have
-been inlineable since their introduction is a `binary-compatible
-source-breaking change`.
-
 Note that if a constant's initial value expression has any observable side
 effects, including the allocation of class instances, it must not be treated
-as inlineable. A constant must always behave as if it is initialized exactly
+as inlinable. A constant must always behave as if it is initialized exactly
 once.
 
 .. admonition:: TODO
 
     Is this a condition we can detect at compile-time? Do we have to be
     restricted to things that can be lowered to compile-time constants?
 
+.. admonition:: TODO
+
+    ``@inlinableAccess`` isn't implemented yet, but for computed properties we
+    already allow putting ``@inlinable`` on the accessors individually. That
```

Comment:
Currently implemented:

- Computed properties can be `@inlinable` (or individual accessors, as you say)
- Stored properties can be `@_fixed_layout` (which mostly does what you describe for `@inlinableAccess`)

Reply:
Yeah. I'm not clear on how these compose, but maybe they don't need to. Stored properties with observers, maybe?